salt/modules/lxd.py

```diff
@@ -688,7 +688,11 @@ def container_create(
     # Add devices if not wait and devices have been given.
     if devices:
         for dn, dargs in devices.items():
-            container_device_add(name, dn, **dargs)
+            if "type" in dargs:
+                # extract 'type' so it won't be overwritten
+                container_device_add(name, dn, dargs["type"], **dargs)
```

Comment:
```suggestion
                container_device_add(name, dn, device_type=dargs["type"], **dargs)
```

Just want to ensure kwargs are kept in tact in case someone changes the order.